Yeni klasör/Lesson16/feyza/script.js

```diff
@@ -0,0 +1,250 @@
+	}
+}
+console.log("Exercise 10", canVote(49));
+
+// 11. Write a function that asks the user to enter their salary and their years of experience. If the salary is less than $30,000 and the years of experience
+// are less than 5, print "Sorry, we cannot offer you the job at this time". If the salary is less than $50,000 and the years of experience are 
+//between 5 and 10 (inclusive), print "We can offer you the job at a lower salary". If the salary is greater than or equal to $50,000 or the years of 
+//experience are greater than or equal to 10, print "We can offer you the job at the requested salary".
+
+function jobOffer(salary,experience){
+	if(salary < 30000 && experience <5){
+		print("Sorry, we cannot offer you the job at this time");
+	}else if(salary < 50000 && experience > 5 && experience < 10){
+		print("We can offer you the job at a lower salary");
+	}else if(salary >= 50000 && experience >= 10){
+		print("We can offer you the job at the requested salary");
+	}
+}
+
+console.log("Exercise 11",jobOffer(48000,7));
+
+// 12. Write a function that asks the user to enter their score on a test. If the score is less than 60, print "You failed the test". 
+//If the score is between 60 and 70 (inclusive), print "You passed the test, but your grade is not great". If the score is between 71 and 80 (inclusive),
+// print "You got a B". If the score is between 81 and 90 (inclusive), print "You got an A". If the score is greater than 90, print "You got an A+ - Great job!".
+
+function isScoreTest(score){
+	if(score < 60){
+		print("You failed the test");
+	}else if(score <= 60 && score >= 70){
+		print("You passed the test, but your grade is not great");
+	}else if(score <= 71 && score >= 80){
+		print("You got a B");
+	}else if(score <=81 && score >= 90){
+		print("You got an A");
+	}else if(score > 90){
+		print("You got an A+ - Great job!");
+	}
+}
+console.log("Exercise 12", isScoreTest(100));
+
+// 13. Write a function that asks the user to enter a month number (1-12) and displays the corresponding season name. Use a switch statement to handle 
+//the different cases.
+
+function getSeasonName(numberOfAMonth) {
+	switch (numberOfAMonth) {
+      // numberOfAMonth === 12
+		case 12:
+		case 1:
+		case 2:
+      // console.log('Winter');
+			// break;
+			return 'Winter';
+		case 3:
+		case 4:
+		case 5:
+			return 'Spring';
+		case 6:
+		case 7:
+		case 8:
+			return 'Summer';
+		case 9:
+		case 10:
+		case 11:
+			return 'Autumn';
+    default: 
+      return 'Value is not supported, please enter value between 1 and 12.';
+	}
+}
+
+console.log('Exercise 13', getSeasonName(1), getSeasonName(5), getSeasonName(20));
+
+// 14. Write a function that asks the user to enter a grade letter (A, B, C, D, or F) and displays a message indicating whether the grade is passing (A, B, C) or 
+//failing (D, F). Use a switch statement to handle the different cases.
+function gradeLetter(letter){
+	letter=letter.toUpperCase();
+	switch (letter) {
+		case A:
```

Comment:
Case values should be a string type as we try to match string here. Otherwise, compiler will try to find variables A,B,C,D,F in the scope.
```suggestion
		case 'A':
```